packages/java/endpoint/src/main/java/com/vaadin/hilla/crud/ListRepositoryService.java

```diff
@@ -52,10 +57,13 @@ public ListRepositoryService(R repository) {
     }
 
     @PostConstruct
-    private void init() {
+    void init() {
         if (repository == null) {
             repository = resolveRepository();
         }
+        var env = applicationContext.getEnvironment();
+        maxPageSize = env.getProperty("spring.data.rest.pageable.max-page-size",
```

Comment:
I this property should be like this:
```suggestion
        maxPageSize = env.getProperty("spring.data.web.pageable.max-page-size",
```
As the one from `spring.data.rest` is:
```
spring.data.rest.max-page-size
```
and users normally expect the latter to work only while using `spring.data.rest` to automatically expose rest endpoints from their repositories.